README.rst

```diff
@@ -36,7 +36,14 @@ Installation
 
 Note that if you've already installed `mesos` locally, you can either install this to a location other than `/usr/local/bin` via. pip options or remove `/usr/local/bin/mesos`. There should be no downsides to just removing it.
 
-To build `mesos-cli`, run `python setup.py build` in the root of the repository. To install, run `python setup.py install`.
+
+To build, run ``python setup.py build`` in the root of the repository.
+
+To install, run ``python setup.py install``
+
+To make the development environment, run ``make env && source env/bin/activate``
```

Comment:
I don't think you will be able to do the `source` part in `make` because it is a subprocess.